app/helpers/users_helper.rb

```diff
@@ -1,9 +1,14 @@
 # frozen_string_literal: true
 
 module UsersHelper
-  def gravatar_for(user)
+  def gravatar_for(user, _options = {})
     gravatar_id = Digest::MD5.hexdigest(user.email.downcase)
-    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}"
+    size = Settings.size_80
+    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?s=#{size}"
```

Comment:
```suggestion
    gravatar_url = "https://secure.gravatar.com/avatar/#{gravatar_id}?s=#{Settings.size_80}"
```
    Bỏ tham số _options ở trên